examples/using-awssns/Readme.md

```diff
@@ -15,9 +15,9 @@ Using Amazon SNS topics, your publisher systems can fanout messages to a large n
 
 To run the example follow the steps below :
 
-1) Run the below command on your PWD to setup awssns locally for testing.
+1) Run the below command from root directory to setup awssns locally for testing.
 
-   > `bash init.sh`
+   > `bash ./examples/using-awssns/init.sh`
```

Comment:
I don't think this has to start with ./examples/using-awsns

Reply:
Running `bash init.sh` directly through checking into example directory will return error since in init.sh file we have `docker-compose -f examples/using-awssns/Docker-Compose.yml up -d`.

Reply:
okay